email-ballerina/commons.bal

```diff
@@ -79,12 +77,12 @@ public type Attachment record {|
 
 # Secure Socket configuration.
 #
-# + certificate - Server certificate
+# + cert - Server certificate
 # + protocol - SSL or TLS protocol
 # + ciphers - Ciper used
 # + verifyHostname - Enable hostname verification
 public type SecureSocket record {|
```

Comment:
This API doesn't match to the convention we are following. Please check https://docs.google.com/document/d/1Y2kLTOw9-sRK1vSEzw5NYdWSA4nwVCvPf3wrbwNDA4s/edit

Also, we can use `optional` fields instead of `nil-able` fields.

Reply:
Changed some fields in the API for the convention.